bot/cogs/snekbox.py

```diff
@@ -212,7 +212,12 @@ async def send_eval(self, ctx: Context, code: str) -> Message:
             else:
                 self.bot.stats.incr("snekbox.python.success")
 
-            response = await ctx.send(msg)
+            filter_cog = self.bot.get_cog("Filtering")
+            filter_triggered = await filter_cog._filter_eval(msg, ctx.message)
```

Comment:
You've misunderstood the issue. The bot's output needs to be filtered, not the invocation message. The latter was already getting caught by the filter.

Reply:
The invocation message is only passed to check for whether to filter the message (as Staff messages aren't filtered) and for user information for when a filter is triggered. The first parameter (`msg`) is the bot's output that's checked before it's sent